homeassistant/components/assist_satellite/entity.py

```diff
@@ -36,11 +46,112 @@ class AssistSatelliteEntity(entity.Entity):
     entity_description: AssistSatelliteEntityDescription
     _attr_should_poll = False
     _attr_state: AssistSatelliteState | None = None
+    _attr_supported_features = AssistSatelliteEntityFeature(0)
 
     _conversation_id: str | None = None
     _conversation_id_time: float | None = None
 
-    _run_has_tts: bool = False
+    _is_announcing: bool = False
+    _tts_finished_event: asyncio.Event | None = None
+    _wake_word_future: asyncio.Future[str | None] | None = None
+
+    async def async_added_to_hass(self) -> None:
+        """Run when entity about to be added to hass."""
+        self._set_state(AssistSatelliteState.LISTENING_WAKE_WORD)
+
```

Comment:
This should be left up to the platforms. We don't know if wake word is being listened to.

```suggestion
```